lib/views/oauth_page.dart

```diff
@@ -153,8 +154,18 @@ class OAuthPageState extends State<OAuthPage> {
       LenraApi.instance.token = response.accessToken;
 
       if (context.read<AuthModel>().user == null) {
-        UserResponse user = await UserApi.me();
-        context.read<AuthModel>().user = user.user;
+        try {
+          UserResponse user = await UserApi.me();
+          context.read<AuthModel>().user = user.user;
```

Comment:
You should rename the var to be clearer:

```suggestion
          UserResponse resp = await UserApi.me();
          context.read<AuthModel>().user = resp.user;
```